src/config/featureFlags/schema.ts

```diff
@@ -20,7 +20,7 @@ export const FeatureFlagsSchema = z.object({
   dalle: z.boolean().optional(),
   speech_to_text: z.boolean().optional(),
   token_counter: z.boolean().optional(),
-
+  model_name: z.boolean().optional(),
```

Comment:
`model_name` is not clear enough for this feature. 

I perfer to `model_tag_display_mode` and with an enum. 

The default value should be `model_id`, and your perference is `model_name` . Maybe we can have another value for `provider/model_id` to show `openai/gpt-4o`

Reply:
But feature flags only support boolean values now, doesn't it?

Reply:
This docs doesn't mention how to set a enum value for a feature flag: https://lobehub.com/docs/self-hosting/advanced/feature-flags

Reply:
> But feature flags only support boolean values now, doesn't it?

it seems yes. But we can extend it just like the sytnax of `OPENAI_MODEL_LIST` : https://lobehub.com/docs/self-hosting/advanced/model-list. 

`+key=value`